packages/headless/src/controllers/core/search-parameter-manager/headless-core-search-parameter-manager.test.ts

```diff
@@ -47,6 +61,25 @@ describe('search parameter manager', () => {
     initSearchParameterManager();
   });
 
+  it('it adds the correct reducers to engine', () => {
```

Comment:
👍 

Out of curiosity, do you know if there's a reason why we usually phrase this `it` description as "it adds ..." rather than just "adds ..."? Total pet peeve, but it does annoy me a tiny bit 😂